src/core_modules/capture-core/components/FiltersForTypes/Date/options.js

```diff
@@ -20,5 +20,13 @@ export const mainOptionTranslatedTexts = {
     [mainOptionKeys.LAST_WEEK]: i18n.t('Last week'),
     [mainOptionKeys.LAST_MONTH]: i18n.t('Last month'),
     [mainOptionKeys.LAST_3_MONTHS]: i18n.t('Last 3 months'),
-    [mainOptionKeys.CUSTOM_RANGE]: i18n.t('Custom range'),
+    [mainOptionKeys.ABSOLUTE_RANGE]: i18n.t('Absolute range'),
+};
+
+export const optionKeys = {
+    RELATIVE_RANGE: 'RELATIVE_RANGE',
+};
+
+export const optionTranslatedTexts = {
+    [optionKeys.RELATIVE_RANGE]: i18n.t('Relative range'),
```

Comment:
Hey, 

The reason for not using mainOptionKeys and mainOptionTranslatedTexts is due to the current limitation of `SelectBoxes` component. At the moment, the component renders for each option only a radio button and a basic text label. 

When I initially worked on this feature I implemented the fastest solution in the hope it will be merged in the `2.38` release. To avoid a complex refactor inside SelectBoxes, I decided to use 2 `SelectBoxes` with 2 different `optionKeys`. 

Now that 2.38 is frozen, I can spend more time to refactor the `SelectBoxes` component and give support to custom labels with custom components within. Let me know if this makes sense and if you have a better suggestion on how to implement it. If you agree with the `SelectBoxes` component refactor should I do it in this PR or another ticket?

Thank you!

Reply:
Agreed on Slack